metrics/README.md

```diff
@@ -25,3 +25,69 @@ function in that file has the ability to also `curl` or `socat` the JSON results
 by environment variables (see the file source for details). This method has been used to store results in
 Elasticsearch and InfluxDB databases for instance, but should be adaptable to use with any REST API that accepts
 JSON input.
+
+## Scaling execution
+This section describes a complete step-by-step scaling execution upto results reporting by using `scaling/k8s_scale.sh` tool which launch a series of workloads and take memory metric measurements after each launch.
+
+**Requirements**
+* A kubernetes cluster up and running (tested on v1.15.3).
+* `bc` and `jq` packages.
+* Docker (only for report generation).
+
+The steps to execute scaling framework are listed below, which need to be executed into the master node of the kubernetes cluster to avoid network issues:
+1. Clone `cloud-native-setup` repository into a preferred directory.
+2. Change directory upto `cloud-native-setup/metris`.
+3. Launch the execution by:
+   ```sh
+   $ ./scaling/k8s_scale.sh
+   INFO: Initialising
+   command: bc: yes
+   command: jq: yes
+   INFO: Checking k8s accessible
+   INFO: 1 k8s nodes in 'Ready' state found
+   starting kubectl proxy
+   Starting to serve on 127.0.0.1:8090
+   daemonset.apps/stats created
+   Waiting for daemon set "stats" rollout to finish: 0 of 1 updated pods are available...
+   daemon set "stats" successfully rolled out
+   INFO: Running test
+   INFO: And grab some stats
+   INFO: idle [98.49] free [29031100] launch [0] node [clr-30f01b5149ba4ab8b05a7ee03b6812a5] inodes_free [31103039]
+   INFO: Testing replicas 1 of 20
+   INFO: Content of runtime_command=:/@RUNTIMECLASS@/d
+   ...
+   ```
+The above execution might take about 4min and it will generate a `k8s-scaling.json` result file at `result` directory.
+
```

Comment:
It may be good to explain why it takes that time. It's because it's only launching [20 pods](https://github.com/clearlinux/cloud-native-setup/blob/master/metrics/scaling/k8s_scale.sh#L21) by default.

Reply:
Done :)